sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -149,6 +149,10 @@ def on_request(self, request):
         request_id = unset = object()
         if 'request_id' in request.context.options:
             request_id = request.context.options.pop('request_id')
+            if not request_id:
```

Comment:
Note (not feedback):
That will work with `request_id=None` as expected, but also `request_id=False` or `request_id=[]`.
I think it's fair, and that would be overreacting to try to deal with that